gulpfile.js

```diff
@@ -97,12 +97,14 @@ gulp.task('update-version', function () {
             dist.js + '*.js'
             ], {
```

Comment:
Could you export this array of globs to a variable above the return? It'll be more readable.